cluster-scope/base/user.openshift.io/groups/open-aiops/group.yaml

```diff
@@ -1,6 +1,10 @@
----
 apiVersion: user.openshift.io/v1
 kind: Group
 metadata:
   name: open-aiops
-users: []
+users:
+  - davidohana
+  - ronensc
+  # FIXME: account missing
```

Comment:
can be removed, was confirmed by a team member

Reply:
updated